### PR TITLE
Rename `has-role` to `relates`

### PR DESCRIPTION
**Pull request by aelred**

### CODE REVIEW

File: grakn-core/src/main/java/ai/grakn/util/Schema.java [hunk 1]
```diff
@@ -49,7 +49,7 @@ private Schema() {
     public enum EdgeLabel {
         ISA("isa"),
         SUB("sub"),
-        HAS_ROLE("has-role"),
+        HAS_ROLE("relates"),
```
Comment (alexandraorth) [inline grakn-core/src/main/java/ai/grakn/util/Schema.java]: The EdgeLabel here is still `HAS_ROLE`

File: grakn-graql/src/main/java/ai/grakn/graql/internal/analytics/GraknVertexProgram.java [hunk 2]
```diff
@@ -141,18 +141,18 @@ void degreeStepInstance(Vertex vertex, Messenger<Long> messenger) {
     }
 
     void degreeStepCasting(Messenger<Long> messenger) {
-        boolean hasRolePlayer = false;
+        boolean relatesPlayer = false;
```
Comment (alexandraorth) [inline grakn-graql/src/main/java/ai/grakn/graql/internal/analytics/GraknVertexProgram.java]: I think this boolean is missing some sort of "has", perhaps "hasPlayer"?

File: grakn-graql/src/main/java/ai/grakn/graql/internal/hal/HALConceptOntology.java [hunk 3]
```diff
@@ -52,7 +52,7 @@
     private final static String ONTOLOGY_LINK = "ontology";
     private final static String OUTBOUND_EDGE = "OUT";
     private final static String INBOUND_EDGE = "IN";
-    private final static String HAS_ROLE_EDGE = "has-role";
+    private final static String HAS_ROLE_EDGE = "relates";
```
Comment (alexandraorth) [inline grakn-graql/src/main/java/ai/grakn/graql/internal/hal/HALConceptOntology.java]: Same as before, should be `RELATES_EDGE`

**Comment by aelred** [comment]
please retest this please - failure with `ai.grakn.test.engine.controller.TasksControllerTest.afterSendingTaskWithRunAt_ItIsDelayedInStorage`

**Comment by aelred** [comment]
please retest this please - failure at `ai.grakn.test.engine.GraknEngineServerTest.whenEnginePropertiesDoesNotIndicateZookeeperStorage_GraphStorageIsUsed`

**Pull request merged**